crates/biome_js_analyze/src/lint/nursery/use_import_extensions.rs

```diff
@@ -82,18 +107,39 @@ declare_rule! {
     }
 }
 
+#[derive(Clone, Debug, Default, Deserializable, Deserialize, Serialize, Eq, PartialEq)]
+#[cfg_attr(feature = "schemars", derive(JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct UseImportExtensionsOptions {
```

Comment:
What do you think about a configuration like this:

```json
{
	"suggestedExtensions": [
		{
			"inputExtensions": ["ts", "tsx"],
			"import": "js",
			"component": "jsx"
		}
	]
}
```

Reply:
That's what I had intially before this comment: https://github.com/biomejs/biome/pull/3274#issuecomment-2186563816.

I'm not really sure which option is better, both seem sufficient to case for handling backend code + `tsc`

Reply:
I have no big preference. However, using a hash map is easier to query in the code. I think we should not aim for perfection here, because the mapping should be a temporary workaround. We should not spend too much time on that.